cmdtests/config_test.go

```diff
@@ -0,0 +1,83 @@
+// +build integration
+
+package cmdtests_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/src-d/engine/cmd/srcd/config"
+	"github.com/src-d/engine/cmdtests"
+	"github.com/stretchr/testify/suite"
+)
+
+type ConfigTestSuite struct {
```

Comment:
Maybe we could also test the command without passing `--config` by overriding the `HOME` env var in `cmd.Env`.

Reply:
:+1: done in 77dadee